modules/UI/UI.js

```diff
@@ -510,8 +510,8 @@ UI.dockToolbar = dock => APP.store.dispatch(dockToolbox(dock));
  * @param {string} avatarURL - The URL to avatar image to display.
  * @returns {void}
  */
-UI.refreshAvatarDisplay = function(id, avatarURL) {
-    VideoLayout.changeUserAvatar(id, avatarURL);
+UI.refreshAvatarDisplay = function(id) {
```

Comment:
I agree in general. If you can reduce code from modules/ better!

Reply:
So it turns out this isn't as easy to kill off right now, I think. Maybe with additional changes it might be, but looking at the branch in its current state, the reason this call is needed is really to tell the local video to re-render its avatar because the local participant got a new id (joined the conference). Without this method, the local video will continue to use some initial default id, leaving the avatar to also use the initial default id instead of the local participant id stored in redux.